src/btree/bt_debug.c

```diff
@@ -433,7 +435,7 @@ __debug_hs_cursor(WT_DBG *ds, WT_CURSOR *hs_cursor)
         WT_RET(ds->f(ds,
           "\t"
           "hs-modify: %s\n",
-          __wt_time_window_to_string(&tw, time_string)));
+          __wt_time_window_to_string(&cbt->modify_update->tw, time_string)));
```

Comment:
I think we should use cbt->upd_value here as well. All the values in hs are full value. The WT_UPDATE_MODIFY case means we are storing reverse modify for the data store in the history store but the history store value itself is a full value. Worth checking with @tetsuo-cpp as I'm not sure what is cbt->modify_update is used for.

Reply:
Agreed. You want to use `cbt->upd_value->tw` here.
I can't remember exactly what `cbt->modify_update` is but I believe it's a temporary structure used in the write path.

Reply:
Thank you @quchenhao and @tetsuo-cpp